src/clusterfuzz/_internal/tests/appengine/libs/issue_filer_test.py

```diff
@@ -826,6 +832,37 @@ def test_crash_type_labels(self):
     self.assertNotIn('Resource Exhaustion',
                      issue_tracker._itm.last_issue.labels)
 
+  def test_additional_crash_subtypes(self):
+    """Test filing of crash_subtypes."""
+    policy = issue_tracker_policy.IssueTrackerPolicy({
+        'status': {
+            'assigned': 'Assigned',
+            'duplicate': 'Duplicate',
+            'verified': 'Verified',
+            'new': 'Untriaged',
+            'wontfix': 'WontFix',
+            'fixed': 'Fixed'
+        },
+        'all': {
+            'status': 'new',
+        },
+        'non_security': {},
+        'labels': {
+            'crash_subtypes': {
```

Comment:
How about crash_categories? subtypes means something different to me from what's being intended here.

Reply:
Thanks Oliver, I've changed this variable name.